feature-utils/poly-look/test/react-components/overlays/sideSheet.test.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { SideSheet } from "../../../src/react-components";
+
+/**
+ * @jest-environment jsdom
+ */
+describe("SideSheet", () => {
+  test("Creates an empty container with close buttons", () => {
+    const res = render(<SideSheet />);
+    expect(res.container).toBeTruthy();
+    expect(res.queryAllByRole("button").length).toBe(2);
+    expect(res.queryAllByRole("h1").length).toBe(0);
+  });
+
+  test("Renders title", () => {
+    const { getByText } = render(<SideSheet title="testing" />);
+    expect(getByText("testing")).toBeTruthy();
+  });
+
+  test("Renders children", () => {
+    const res = render(
```

Comment:
Nit: For consistency you could also destructure `getByText` here, hm? But totally unimportant.